src/app/data/token-atm-configuration.ts

```diff
@@ -140,6 +144,14 @@ export class TokenATMConfiguration {
         };
     }
 
+    // Warning: This function will override previous secure config!
+    public regenrateSecureConfig(): void {
+        this.#encryptionKey = undefined;
+        // https://stackoverflow.com/a/47496558
+        this.#encryptionPassword = [...Array(32)].map(() => Math.random().toString(36)[2]).join('');
```

Comment:
Nice catch! I will still use `0` instead of `@` for now since generated strings are saved in a raw form in HTML, while `@` should be replaced by `&commat;` We should consider the use of `crypto.getRandomValues` later to further enhance the security.